Formula/run-one.rb

```diff
@@ -0,0 +1,23 @@
+class RunOne < Formula
+  desc "Run one instance of a unique combination of command and args at a time"
+  homepage "https://launchpad.net/run-one"
+  url "https://github.com/datamade/run-one/archive/ea3d7744043bf737aa52c3093f068a49b7621d02.tar.gz"
+  version "1.18"
+  sha256 "8094b26db3efca2a419737ab26d1926e98c116d9e687dc32cd841192c3192dfc"
+  depends_on "flock"
```

Comment:
We don't have a `flock` formula in Homebrew core.

Reply:
Sorry I forgot that the dependency was only available through a cask. I'll withdraw this PR. https://github.com/discoteq/flock